jsbits/diff.js

```diff
@@ -70,13 +84,11 @@ function diffProps (cProps, nProps, node, isSvg) {
 	     if (isSvg || !(c in node))
 	       node.removeAttribute(c, cProps[c]);
 	     else
-	       node[c] = "";
+    	       node[c] = '';
```

Comment:
Some of these lines have both tab and space characters for indentation. It's not because of this PR, but is it intentional? It feels like some of the indentation fixes are because of different tab sizes in different editors.